cmd/pctl/main.go

```diff
@@ -37,16 +41,16 @@ func searchCmd() *cli.Command {
 	return &cli.Command{
 		Name:      "search",
 		Usage:     "search for a profile",
-		UsageText: "pctl --catalog-url <URL> search <QUERY>",
+		UsageText: "pctl --kubeconfig=<kubeconfig-path> search <QUERY>",
 		Action: func(c *cli.Context) error {
-			searchName, catalogURL, err := parseArgs(c)
+			searchName, catalogClient, err := parseArgs(c)
 			if err != nil {
 				_ = cli.ShowCommandHelp(c, "show")
```

Comment:
```suggestion
				_ = cli.ShowCommandHelp(c, "search")
```
nothing to do with this PR, but can I sneak this in :p

Reply:
sure